### PR TITLE
Fixes all ghost spawners being visible always

### PR DESCRIPTION
**Pull request by DreamySkrell**
😩

### CODE REVIEW

**Comment by DreamySkrell** [comment]
!bugfix

**Comment by DreamySkrell** [comment]
!review

File: code/modules/ghostroles/spawner/base.dm [hunk 1]
```diff
@@ -18,14 +18,24 @@
...
-	var/enable_dmessage = TRUE //The message to send to deadchat if the ghostspawner is enabled or TRUE for a default message
-	var/respawn_flag = null //Flag to check for when trying to spawn someone of that type (CREW, ANIMAL, MINISYNTH)
+	// If the spawnpoint is enabled
+	var/enabled = TRUE
+	// If set to a value other than null, has the set chance to become enabled
+	var/enable_chance = null
+	// The message to send to deadchat if the ghostspawner is enabled or TRUE for a default message
+	var/enable_dmessage = TRUE
+	// Flag to check for when trying to spawn someone of that type (CREW, ANIMAL, MINISYNTH)
+	var/respawn_flag = null
```
Comment (FluffyGhoster) [inline code/modules/ghostroles/spawner/base.dm:36]: DMDoc these, three /

Comment (DreamySkrell) [reply code/modules/ghostroles/spawner/base.dm]: i forgor

**Review by Alberyk — APPROVED** [review]

**Review by Arrow768 — APPROVED** [review]

**Pull request merged**